### PR TITLE
wunter8: grabbing csrf token/renaming sid file and debugging

### PR DESCRIPTION
**Pull request by leogomezz4t**

### CODE REVIEW

**Comment by leogomezz4t** [comment]
If you want to test it out please post your results here. I'll merge this into the main branch soon

File: pytextnow/TNAPI.py [hunk 1]
```diff
@@ -108,24 +108,26 @@ def get_initial_csrf_token(self):
...
+            with open(self._user_cookies_file, "w") as user_cookies_file:
+                user_cookies_file.write(json.dumps(user_cookies))
 
             self.__init__(self.username, cookie)
         else:
             if cookie:
-                user_SIDS[self.username] = cookie
-                with open(self._user_sid_file, "w") as user_SIDS_file:
-                    user_SIDS_file.write(json.dumps(user_SIDS))
+                user_cookies[self.username] = cookie
```
Comment (wunter8) [inline pytextnow/TNAPI.py:128]: `cookie` in this case will have to be a dict with 'sid' and 'csrf' keys. Do you want to change the method to accept an sid_cookie and a csrf_cookie separately like I did with __init__? Or will you just expect the user to pass in a dict?

Comment (leogomezz4t) [reply pytextnow/TNAPI.py:128]: Oh that was a mistake lol thnx for pointing it out

Comment (leogomezz4t) [reply pytextnow/TNAPI.py]: Alright I pushed a new commit

File: pytextnow/TNAPI.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
     from pytextnow.multi_media_message import MultiMediaMessage
     from pytextnow.message import Message
 import mimetypes
+from re import M
```
Comment (wunter8) [inline pytextnow/TNAPI.py:10]: Was this for something I changed?

Comment (leogomezz4t) [reply pytextnow/TNAPI.py:10]: Oh no my editor must of added that automatically

Comment (leogomezz4t) [reply pytextnow/TNAPI.py:10]: alright is there anything else I mightve missed and ill group it all into one commit

File: pytextnow/TNAPI.py [hunk 3]
```diff
@@ -28,16 +29,16 @@
 class Client:
     def __init__(self, username: str = None, sid_cookie=None, csrf_cookie=None):
         # Load SIDS
-        self._user_sid = {}
+        self._user_cookies = {}
         self._good_parse = False
-        self._user_sid_file = join(dirname(realpath(__file__)), 'user_sids.json')
+        self._user_cookies_file = join(dirname(realpath(__file__)), 'user_sids.json')
```
Comment (wunter8) [inline pytextnow/TNAPI.py:34]: Filename needs to be changed to `user_cookies.json`

Comment (leogomezz4t) [reply pytextnow/TNAPI.py:34]: thnx

**Comment by wunter8** [comment]
Looks good to me

**Pull request merged**